pages/price-feeds/use-real-time-data/solana.mdx

```diff
@@ -189,7 +203,7 @@ await transactionBuilder.addPriceConsumerInstructions(
   }
 );
 
-// Send the instructions in the builder in 1 or more transactions.
+// Send the instructions to the builder in 1 or more transactions.
```

Comment:
The old text is more accurate, we have already added to the instruction to the builder at this point, and we just need to send them.